Templates/integration_template.ps1

```diff
@@ -0,0 +1,97 @@
+# Template for Integration Testing
+
+# Copy to Program Files for WMF 4.0 Compatability as it can only find resources in a few known places.
+$DSCModuleName = 'xNetworking'
+$DSCResourceName = 'x<ResourceName>'
```

Comment:
I was just working on these templates as part of Issue #47 and have set a +++ to indicate that. But I'll convert it to TODO as I that is the global "Fix me up" :+1: